Formula/benerator.rb

```diff
@@ -0,0 +1,41 @@
+class Benerator < Formula
+  desc "Tool for realistic test data generation"
+  homepage "https://rapiddweller.github.io/homebrew-benerator/"
+  url "https://github.com/rapiddweller/rapiddweller-benerator-ce/releases/download/3.1.0/rapiddweller-benerator-ce-3.1.0-jdk-11-dist.tar.gz"
+  sha256 "194feb051ae18cfcd407b8e1668ce9c60561394bc454f9fc9747c274166843bc"
+
+  license "Apache-2.0"
+  depends_on "openjdk@11"
```

Comment:
Can't it work with a more modern version?

Reply:
Yes, it should work with version 11 and higher
```suggestion
  depends_on "openjdk" => "11"
```

Reply:
That's not valid syntax

Reply:
Yes, that's fine